Yank/repex.py

```diff
@@ -1162,1633 +1507,482 @@ def _display_citations(self):
         print("Please cite the following:")
         print("")
         print(openmm_citations)
-        if self.replica_mixing_scheme == 'swap-all':
+        if self._replica_mixing_scheme == 'swap-all':
             print(gibbs_citations)
-        if self.online_analysis:
-            print(mbar_citations)
-
-        return
-
-    def _create_context(self, system, integrator):
-        """
-        Shortcut to handle creation of a context with or without user-selected
-        platform.
-
-        Parameters
-        ----------
-        system : simtk.openmm.System
-           The system associated to the context.
-        integrator : simtk.openmm.Integrator
-           The integrator to use for Context creation.
 
-        Returns
-        -------
-        context : simtk.openmm.Context
-           The created OpenMM Context object.
+    # -------------------------------------------------------------------------
+    # Internal-usage: Initialization and storage utilities.
+    # -------------------------------------------------------------------------
 
-        """
-        if self.platform is None:
-            return self.mm.Context(system, integrator)
-        else:
-            return self.mm.Context(system, integrator, self.platform)
+    @staticmethod
+    def _does_file_exist(file_path):
+        """Check if there is a file at the given path."""
+        return os.path.exists(file_path) and os.path.getsize(file_path) > 0
```

Comment:
I think this is for the case when something went wrong in between opening the file and write the first data. Probably irrelevant though.